test/simple.spec.ts

```diff
@@ -231,17 +231,19 @@ test('can build a schema', () => {
       sliceStart = Math.max(sliceEnd - args.last, 0);
     }
 
+    const edges = array.slice(sliceStart, sliceEnd).map((char) => ({
+      cursor: char.id,
+      node: char,
+    }));
+
     return {
-      edges: array.slice(sliceStart, sliceEnd).map((char) => ({
-        cursor: char.id,
-        node: char,
-      })),
-      pageInfo: {
-        endCursor: array[array.length - 1].id,
+      edges,
+      pageInfo: edges.length > 0 ? {
+        endCursor: edges[edges.length - 1].cursor,
```

Comment:
In the case of an empty list, I tend to use `""` as the start/end cursor values, as the `pageInfo` field is non-nullable per spec.

___

Furthermore, ID != cursor, so it might make sense to to have some kind of `cursorFromRecord` function. IMHO this can be part of a follow-up PR.

Reply:
You're just giving me opinions. 

The spec clearly says what must happen?